data_structures/stacks/evaluate_postfix_notations.py

```diff
@@ -17,9 +17,29 @@ def evaluate_postfix(postfix_notation: list) -> int:
     9
     >>> evaluate_postfix(["4", "13", "5", "/", "+"])
     6
+    >>> evaluate_postfix(["2", "+"])
+    2
+    >>> evaluate_postfix(["5", "-"])
+    -5
     >>> evaluate_postfix([])
     0
-    """
+    >>> evaluate_postfix(["5"])
+    5
+    >>> evaluate_postfix(["5", "A"])
+    Traceback (most recent call last):
+     ...
+    ValueError: invalid literal for int() with base 10: 'A'
+    >>> evaluate_postfix(["5", "4", "A"])
+    Traceback (most recent call last):
+     ...
+    ValueError: invalid literal for int() with base 10: 'A'
+    >>> evaluate_postfix(["A"])
+    Traceback (most recent call last):
+     ...
+    ValueError: invalid literal for int() with base 10: 'A'"""
+    if len(postfix_notation) == 1 and postfix_notation[0] in {"+", "-", "*", "/"}:
+        raise ValueError("Invalid expression: insufficient operands")
+
```

Comment:
```suggestion
```
This can be handled by simplifying logic elsewhere. Plus, listing out the set of operators is redundant with later code.